src/components/InventoryPage/DonationModal.jsx

```diff
@@ -0,0 +1,405 @@
+                <Text mt="60px" mb={5} fontSize="20px">
+                  Address
+                </Text>
+                <Stack spacing={3} direction="row">
+                  <InputGroup>
+                    <InputLeftAddon>Street Address</InputLeftAddon>
+                    <Input
+                      placeholder="street"
+                      defaultValue={street}
+                      onChange={({ target }) => {
+                        setStreet(target.value);
+                      }}
+                      isDisabled={!canEditInfo}
+                    />
+                  </InputGroup>
+                  <InputGroup>
+                    <InputLeftAddon>Unit</InputLeftAddon>
+                    <Input
+                      placeholder="unit"
+                      defaultValue={unit}
+                      onChange={({ target }) => {
+                        setUnit(target.value);
+                      }}
+                      isDisabled={!canEditInfo}
+                    />
+                  </InputGroup>
+                </Stack>
+                <Stack spacing={3} direction="row" my={2}>
+                  <InputGroup>
+                    <InputLeftAddon>City</InputLeftAddon>
+                    <Input
+                      placeholder="city"
+                      defaultValue={city}
+                      onChange={({ target }) => {
+                        setCity(target.value);
+                      }}
+                      isDisabled={!canEditInfo}
+                    />
+                  </InputGroup>
+                  <InputGroup>
+                    <InputLeftAddon>State</InputLeftAddon>
+                    <Input placeholder="state" isDisabled={!canEditInfo} />
+                  </InputGroup>
+                  <InputGroup>
+                    <InputLeftAddon>Zip Code</InputLeftAddon>
+                    <Input
+                      placeholder="zip"
+                      defaultValue={zip}
+                      onChange={({ target }) => {
+                        setZip(target.value);
+                      }}
+                      isDisabled={!canEditInfo}
+                    />
+                  </InputGroup>
+                </Stack>
+                <Text mt="60px" mb={5} fontSize="20px">
+                  Additional Comments
+                </Text>
+                <Textarea
+                  placeholder="Enter additional comments here"
+                  defaultValue={newnote}
+                  onChange={({ target }) => {
+                    setNotes(target.value);
+                  }}
+                  isDisabled={!canEditInfo}
+                />
+              </Box>
+              <Box h={600} w="40%" m={5}>
+                <Box>
+                  <Text mb={5} fontSize="20px">
+                    Images
+                  </Text>
+                  <Box h={300} w={395} bg="gray">
+                    {image ? image.imageUrl : 'no image provided'}
```

Comment:
Were you able to hook this up to open the Image modal on top of the Donation modal?